hail/python/hail/vds/functions.py

```diff
@@ -82,21 +82,18 @@ def local_to_global(array, local_alleles, n_alleles, fill_value, number):
     -------
```

Comment:
We should have an example that uses the fields of a VDS. Do we have a VDS in the tests we can use for that purpose?